src/txmempool.cpp

```diff
@@ -496,7 +496,7 @@ void CTxMemPool::addUnchecked(const CTxMemPoolEntry &entry, setEntries &setAnces
     CAmount delta{0};
     ApplyDelta(entry.GetTx().GetHash(), delta);
     if (delta) {
-            mapTx.modify(newit, update_fee_delta(delta));
+        mapTx.modify(newit, update_modify_fee(delta));
```

Comment:
Since you're touching anyway, might as well update this to use a lambda (now that we're not pre-C++11 anymore), and we can delete the `update_modify_fee` struct. Similar to e177fcab3831b6d259da5164cabedcc9e78f6957
```suggestion
        mapTx.modify(newit, [&delta](CTxMemPoolEntry& e) { e.UpdateModifyFee(delta); });
```

(Same suggestion to L935)

Reply:
Thx, already fixed in master.